apps/lib/s_cb.c

```diff
@@ -894,7 +894,11 @@ int generate_stateless_cookie_callback(SSL *ssl, unsigned char *cookie,
     buffer = app_malloc(length, "cookie generate buffer");
 
     memcpy(buffer, &port, sizeof(port));
-    BIO_ADDR_rawaddress(peer, buffer + sizeof(port), NULL);
+    if (!BIO_ADDR_rawaddress(peer, buffer + sizeof(port), NULL)) {
+        BIO_printf(bio_err, "Failed getting peer address\n");
+        BIO_ADDR_free(lpeer);
```

Comment:
Maybe it would be better to `goto end` as we are missing `OPENSSL_free(buffer);` here

Reply:
Thank you; fixed